### ISSUE TITLE
How can I use CTC loss function ?

### ISSUE THREAD

**Issue by Zerith** [pytorch/pytorch#504]
I want to use pytorch for a usecase where I need the CTC loss function.
How can I use the CTC loss function from [warp-ctc](https://github.com/baidu-research/warp-ctc) in pytorch?

**Comment by apaszke** [comment]
Right now there are no pytorch bindings for CTC. If you'd like to write one, you'd have to transpile [this wrapper file](https://github.com/baidu-research/warp-ctc/blob/master/torch_binding/binding.cpp) into [a pytorch C extension](https://github.com/pytorch/extension-ffi).

**Comment by soumith** [comment]
@SeanNaren has a functional deepspeech2.pytorch with CTC bindings. You can co-ordinate with him. I am going to close out this issue now.

**Comment by SeanNaren** [comment]
I'll be opening it soon, but @apaszke is right. ATM I've just wrapped the numpy API for use, but I'll be working on an actual C extension to use the C api directly
